pegasus/helpers/page_helpers.rb

```diff
@@ -29,3 +29,12 @@ def partner_site?()
   partner_sites = CDO.partners.map{|x|x + '.code.org'}
   return partner_sites.include?(request.site)
 end
+
+def under_13?
+  under_13 = false
+  if dashboard_user
+    age = ((Date.today - dashboard_user[:birthday]) / 365).to_i
+    under_13 = true if age < 13
+  end
+  under_13
+end
```

Comment:
Oh much nicer.

Well we don't hide the petition right now for anyone. And I think it's good enough to start hiding it for users we know are 13, but otherwise don't care. Should I rename to current_user_under_13? so that it's known it's defined a specific way?